Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -3366,6 +3383,11 @@ def visit_SimpleCallNode(self, node):
                 node.function = ExprNodes.NameNode(node.pos, name=EncodedString('set'))
             elif function == u'staticmethod':
                 node.function = ExprNodes.NameNode(node.pos, name=EncodedString('staticmethod'))
+            elif function == u'dataclass':
+                from .Dataclass import make_dataclass_module_callnode
+                dataclass_module = make_dataclass_module_callnode(node.pos)
```

Comment:
Untested code? It says `make_dataclasses_module_callnode` everywhere else.
```suggestion
                from .Dataclass import make_dataclasses_module_callnode
                dataclass_module = make_dataclasses_module_callnode(node.pos)
```

Reply:
I think this was an attempt to present `cython.dataclasses.dataclass` as `dataclasses.dataclass` if it wasn't used in a Cython-specific context - it never worked hugely reliably, and I don't think it's a necessary thing to do, so I'll remove it.